pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -124,6 +125,8 @@ type OperationExecutor interface {
 func NewOperationExecutor(
 	operationGenerator OperationGenerator) OperationExecutor {
 
+	util.RegisterMetrics()
```

Comment:
The volume operation metrics in such a place that I think, there is no good one place to register them except the `util/metrics.go` file itself.   In many cases - we have well defined set of places where registered metrics will be used. Such as - we know that aws metrics will be used in AWS package after initialization. 

We don't have such clear cut isolation in case of volume metrics.  So while I understand the precedent of using `sync.Once` - I do not think we need to bent backwards to use it here. We can get away by simply registering the metrics in `util/metrics` package itself. It would be clean and DRY - wouldn't require `sync.Once`. Is there a downside of registering the metrics in package itself that I am missing?